GVFS/GVFS.Common/NuGetUpgrade/NuGetUpgrader.cs

```diff
@@ -467,33 +462,26 @@ protected static EventMetadata CreateEventMetadata(Exception e = null)
             return metadata;
         }
 
-        private static bool TryGetPersonalAccessToken(string gitBinaryPath, string credentialUrl, ITracer tracer, out string token, out string error)
+        private static string replacementToken(string tokenString)
```

Comment:
??? Lower case function name? this is also a pretty small method. Why not inline?

Reply:
Good catch - this was a lower case string already, and is just showing up in the diff due to things moving around... I will update the casing, but in a different change.